src/themes/default/pages/Static.vue

```diff
@@ -59,17 +57,25 @@ export default {
       this.activeComponent = component
     }
   },
+  watch: {
+    '$route.name': {
+      immediate: true,
+      handler: function () {
+        this.activeComponent = this.navigation.find(nav => nav.link === this.$route.path) ? this.navigation.find(nav => nav.link === this.$route.path).component : null
```

Comment:
why not make active component as computed? We wouldn't need this watcher. 
Also please not use twice 'find'

Reply:
it works exactly the same since it's just a watcher under the hood. but CC looks cleaner so it might be better to change it, thanks :)